src/AspNetCoreModuleV2/AspNetCore/Inc/aspnetcore_shim_config.h

```diff
@@ -105,6 +106,14 @@ class ASPNETCORE_SHIM_CONFIG : IHttpStoredContext
         return m_hostingModel;
     }
 
+    STRU*
```

Comment:
please change the return type as PCWSTR as the caller really needs is the string pointer and you don't want caller to change the string value

Reply:
All other calls in the shim_config return STRU*. I agree with you, but we should change the rest of them too if that is the case.